axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankorder/BankOrderServiceImpl.java

```diff
@@ -280,6 +285,10 @@ public void validatePayment(BankOrder bankOrder) throws AxelorException {
         }
       }
     }
+
+    bankOrderValidatedEvent
+        .select(NamedLiteral.of("validatePayment"))
```

Comment:
Please define constants String in BankOrderValidated instead of using a hard coded value here.